### PR TITLE
Observation API changes

### PR DESCRIPTION
**Pull request by aks30**

### CODE REVIEW

File: module/observationSubmissions/helper.js [hunk 1]
```diff
@@ -465,7 +465,8 @@ module.exports = class ObservationSubmissionsHelper {
                 "completedDate",
                 "ratingCompletedAt",
                 "observationInformation.name",
-                "observationId"
+                "observationId",
+                "scoringSystem"
```
Comment (aks30) [inline module/observationSubmissions/helper.js:469]: @rakeshSgr - Maybe include one isRubricDriven key as well

File: controllers/v1/observationsController.js [hunk 2]
```diff
@@ -847,10 +847,11 @@ module.exports = class Observations extends Abstract {
                 }
 
                 let assessment = {};
-
+                
                 assessment.name = solutionDocument.name;
                 assessment.description = solutionDocument.description;
                 assessment.externalId = solutionDocument.externalId;
+                assessment.pageHeading = solutionDocument.pageHeading;
```
Comment (aks30) [inline controllers/v1/observationsController.js:854]: @rakeshSgr - Might as well do in v2 and also in assessments controller as well

File: models/solutions.js [hunk 3]
```diff
@@ -108,6 +108,10 @@ module.exports = {
           index : true
         }
       }]
+    },
+    pageHeading: {
+      default : "Domain",
```
Comment (aks30) [inline models/solutions.js:113]: @rakeshSgr "Domains"

File: module/observationSubmissions/helper.js (hunk 1 above)
Comment (rakeshSgr) [reply module/observationSubmissions/helper.js]: resolved

File: controllers/v1/observationsController.js (hunk 2 above)
Comment (rakeshSgr) [reply controllers/v1/observationsController.js:853]: resolved

File: models/solutions.js (hunk 3 above)
Comment (rakeshSgr) [reply models/solutions.js]: resolved

**Pull request merged**